### PR TITLE
Convert test

### PR DESCRIPTION
**Pull request by jens-erdmann**
Convert existing bash tests into pytest. See commits for details.

### CODE REVIEW

File: tests/test_compat.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# SPDX-FileCopyrightText: 2021 Jens Erdmann
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+import sys
+import unittest
+
+TEST_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```
Comment (priv-kweihmann) [inline tests/test_compat.py]: As mentioned earlier already, I would like this sys.path manipulation to be move into a class function

```python
@classmethod
def setup_class(cls):
    sys.path.insert(0, os.path.abspath(os.path.dirname(__file__) + "/../"))

    try:
        from flict.flictlib import flict_config
        from flict.impl import FlictImpl
    except ImportError as e:
        self.fail("Can't import flict... {msg}".format(e))
```

**Review by priv-kweihmann — CHANGES_REQUESTED** [review]
Please update the sys.path manipulation, otherwise I'm good with this change - definitely this is a move in the right direction

File: tests/test_compat.py (hunk 1 above)
Comment (jens-erdmann) [reply tests/test_compat.py]: Should be fixed by now.

File: tests/test_outbounds.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+import pytest
+
+from flict.impl import FlictImpl
+from tests.args_mock import ArgsMock
+
+def _test_expression(expression, result):
```
Comment (priv-kweihmann) [inline tests/test_outbounds.py:11]: Can we move that function to a common class/file? I already see code duplication here

Comment (jens-erdmann) [reply tests/test_outbounds.py:11]: Not really because it calls a different method of FlictImpl in each test.

**Review by priv-kweihmann — APPROVED** [review]

**Pull request merged**